### ISSUE TITLE
Cannot read properties of null (reading 'document')

### ISSUE THREAD

**Issue by 0xdeepmehta** [chantouchsek/vue-iframes#63]
```  
<vue-iframe
          v-if="renderCntType === 'DOC'"
          :src="resourceRenderSrc"
          allow="camera *; geolocation *; microphone *; autoplay *"
          frame-id="my-ifram"
          name="my-frame"
          @load="onLoadFrameDo" 
/>

methods: {
    onLoadFrameDo(frame) {
      this.myIframe = frame.contentWindow
    },
}

nuxt.config.js
plugins: [
    { src: '~/plugins/vue-iframes', mode: 'client'}
  ],
```
Frequently, It is throwing this error :(
any suggestion or solution
<img width="600" alt="Screenshot 2021-11-23 at 7 14 35 PM" src="https://user-images.githubusercontent.com/65382963/143035298-ab3c14ed-a834-43b5-853a-1e008e5ec83e.png">

**Comment by sctouch** [comment]
@0xdeepmehta  did you try to use.
<client-only><vue-iframe /></client-only>

**Comment by chantouchsek** [comment]
Are you trying SSR?

**Comment by 0xdeepmehta** [comment]
> 

yeah I did, It error remain same

**Comment by 0xdeepmehta** [comment]
> @0xdeepmehta did you try to use. `<client-only><vue-iframe /></client-only>`

yeah, I did this also, but error remain their

**Comment by chantouchsek** [comment]
@0xdeepmehta  do you have a small repo to reproduce it?

**Comment by chantouchsek** [comment]
@0xdeepmehta  i don't see any problem, i tested. 

By the way, you can use now `vue-iframes/nuxt`. with latest version.

**Comment by chantouchsek** [comment]
Fixed in [v0.0.19](https://github.com/chantouchsek/vue-iframes/releases/tag/v0.0.19)
